docs/guides/web-ui/dashboards.md

```diff
@@ -35,77 +44,125 @@ Both variants include the following metrics:
+
+You can view and interact with standard dashboards, but you cannot edit them.
+
+### Using a Standard Dashboard as a Template
+
+You can use any standard dashboard as a template by exporting it and then importing it as a custom dashboard.
+
+1. From a standard dashboard, click the **Download dashboard as code** icon in the toolbar on the top right.
+![Download dashboard as code](../../images/guide/browser-download-dashboard-as-code.png)
+2. Go to the **Custom** tab and select the **Import JSON** option.
+3. Import the file you downloaded. This creates a new, fully editable custom dashboard from the template.
+
+---
+
+## Creating custom dashboards
+
+To create a dashboard from scratch:
+
+1. Click the **+ Dashboard** button.
+2. Select the **Custom** tab.
+
+Custom dashboards are structured in a hierarchy: a dashboard contains one or more **panel groups**, and each group contains **panels**. Each panel, in turn, holds a specific **chart type**. By default, new dashboards start with one panel group.
+
+You can add more panel groups to better organize your dashboard. This is useful for grouping related visualizations, effectively allowing you to have multiple views within a single dashboard.
+
+To add a new group, click the **Panel Group** button in the top right. You can name the group and set whether it should be expanded or collapsed by default when the dashboard loads.
+
+To add a new visualization, you add a panel to a group. Click the **Panel** button in the top right. Inside each panel, you'll configure a chart and the SQL query that powers it.
+
+You can rearrange and resize panels by dragging and dropping them after clicking the **Edit layout** button.
+
+### Chart Types
 
-To create a custom dashboard, follow these steps:
+Logfire supports these chart types:
 
-1. From the dashboard page, click on the "Start From Scratch" button.
-3. Once your dashboard is created, you can start rename it and adding charts and blocks to it.
-4. To add a chart, click on the "Add Chart" button.
-5. Choose the type of block you want to add.
-6. Configure the block by providing the necessary data and settings (check the next section).
-7. Repeat steps 4-6 to add more blocks to your dashboard.
-8. To rearrange the blocks, enable the "Edit Mode" in the dashboard setting and simply drag and drop them to the desired position.
+| Chart Type  | Query Type         | Description                          |
+| ----------- | ------------------ | ------------------------------------ |
+| Time Series | TimeSeriesQuery    | Data points over time                |
+| Gauge       | TimeSeriesQuery    | Shows current value as a gauge       |
+| Table       | NonTimeSeriesQuery | Tabular data with rows and columns   |
+| Bar Chart   | NonTimeSeriesQuery | Comparisons across categories        |
+| Pie Chart   | NonTimeSeriesQuery | Data as slices of a circle |
+| Values      | NonTimeSeriesQuery | Values displayed as tiles |
 
```

Comment:
Graphics to demonstrate the types could be lovely - IDK maybe we make 1 graphic with each type in a grid? I can help if you want to collaborate on this? 

![different-types-of-charts-and-graphs-set-column-pie-area-line-graphs-data-analysis-financial-report-business-analytics-illustration-infographic-statistics-graph-vector](https://github.com/user-attachments/assets/c9e42f64-8d0c-4622-a6bc-9378336250da)